pkgs/development/tools/misc/sqitch/default.nix

```diff
@@ -11,7 +11,9 @@ stdenv.mkDerivation {
   installPhase = ''
     mkdir -p $out/bin
     for d in bin/sqitch etc lib share ; do
-      ln -s ${sqitchModule}/$d $out/$d
+      if [ -e ${sqitchModule}/$d ]; then
```

Comment:
Can you add a comment or something? Otherwise it is not obvious why a condition is here.

Reply:
Yeah sure. The point is to prevent a share symlink from being created when it’s destination doesn’t exist (resulting in a collision).

Reply:
Yes, just as a future reference, that this happens on darwin.

Reply:
I'm pretty sure this is not darwin specific.